pages/common/gh-auth.md

```diff
@@ -0,0 +1,24 @@
+# gh-auth
+
+> Authenticate with GitHub in GitHub cli.
+> More information: <https://cli.github.com/manual/gh_auth_login>.
+
+- Login with interactive prompt:
+
+`gh auth login`
+
+- Login with token from https://github.com/settings/tokens :
+
+`echo {{your_token}} | gh auth login --with-token`
+
+- Check if you are logged in:
+
+`gh auth status`
+
+- Log out:
+
+`gh auth logout`
+
+- Login to specific Github Enterprise Server:
+
+`gh auth login --hostname {{enterprise.server.address}}`
```

Comment:
I'm not sure if `--hostname` should be a URL or could be a server name too. Based on `gitlab-runner`:
```suggestion
`gh auth login --hostname {{github.example.com}}`
```

Reply:
I don't have an enterprise server myself, but basing on the gh documentation, they just say "enterprise.internal", so I guess it's a domain name - not URL

Reply:
I'll commit this, since example.com is some super-duper legally-reserved domain for examples - if someone pastes that in without thinking :shrug: